python_modules/dagster/dagster/core/types/builtin_config_schemas.py

```diff
@@ -13,13 +14,14 @@ def define_path_dict_field():
 def define_builtin_scalar_output_schema(scalar_name):
     check.str_param(scalar_name, 'scalar_name')
 
-    class _MaterializeableBuiltinScalarConfigSchema(ConfigSelector):
-        def __init__(self):
-            super(_MaterializeableBuiltinScalarConfigSchema, self).__init__(
-                name=scalar_name + '.MaterializationSchema',
-                description='Materialization schema for scalar ' + scalar_name,
-                fields={'json': define_path_dict_field()},
-            )
+    schema_cls = NamedSelector(
+        scalar_name + '.MaterializationSchema', {'json': define_path_dict_field()}
```

Comment:
I think the time is coming when we will have to revisit the system-generated names

Reply:
yup